docs/run_amd_gpus.rst

```diff
@@ -0,0 +1,37 @@
+.. _run_fasteddy_amdgpu:
+
+**************************
+Building FastEddy on AMD GPU Accelerated platforms
+**************************
```

Comment:
```suggestion
**************************************************
```

Reply:
@fluidnumerics-joe Section headers need to be at least as long as the text (for reference https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections)